### ISSUE TITLE
`selector-list-comma-newline-after` fails with SCSS `//` comment

### ISSUE THREAD

**Issue by deepfriedmind** [stylelint/stylelint#758]
The following triggers `Expected newline after "," (selector-list-comma-newline-after)`:

```scss
a, // comment
b { color: pink; }
```

I'm guessing [indextoCheckAfter](https://github.com/stylelint/stylelint/blob/master/src/rules/selector-list-comma-newline-after/index.js#L38) could be slightly tweaked to allow for this?

**Comment by jeddy3** [comment]
Released in `4.4.0`.

**Comment by deepfriedmind** [comment]
Excellent!
